packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -2,4 +2,51 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-class DeepLinksController {}
+import 'package:flutter/foundation.dart';
+
+import 'deep_links_model.dart';
+import 'fake_data.dart';
+
+
+class DeepLinksController {
+
+  final _linkDatasNotifier = ValueNotifier<List<LinkData>>(allLinkDatas);
+  ValueListenable<List<LinkData>> get linkDatasNotifier => _linkDatasNotifier;
```

Comment:
style nit: order getters before private variables: https://github.com/flutter/devtools/blob/master/packages/README.md#order-of-getters-and-setters. Here and elsewhere